scripts/start-workers.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# EPFL Bigdata Course 2015, remote actor starter/stopper script V.1
+
+# Start and stop the remote actor system on our worker nodes
+
+if [ $# -ne "1" ]; then
+	echo "Usage: ./start-workers.sh <start | stop | status>"
```

Comment:
Rename script to reflect the fact that we can also stop / get status with the same script (e.g. Apache has `apachectl` I think).